spec/stat_tracker_spec.rb

```diff
@@ -34,20 +36,69 @@
   end
 
   it 'returns count of games by season' do
-    expect(stat_tracker.count_of_games_by_season).to eq({'20122013' => 7})
+    expect(stat_tracker.count_of_games_by_season).to eq({ '20122013' => 7 })
   end
 
   it 'returns average goals per game' do
     expect(stat_tracker.average_goals_per_game).to eq(4.29)
   end
 
   it 'returns total goals by season' do
-    expect(stat_tracker.total_goals_by_season).to eq({'20122013' => 30})
+    expect(stat_tracker.total_goals_by_season).to eq({ '20122013' => 30 })
   end
 
   it 'returns average goals by season' do
-    expect(stat_tracker.average_goals_by_season).to eq({'20122013' => 4.29})
+    expect(stat_tracker.average_goals_by_season).to eq({ '20122013' => 4.29 })
+  end
+
+  it 'returns count of teams' do
+    expect(stat_tracker.count_of_teams).to eq(32)
+  end
+
+  it 'returns team with best offense' do
+    expect(stat_tracker.best_offense).to eq("FC Dallas")
+  end
+
+  it 'returns team with worst offense' do
+    expect(stat_tracker.worst_offense).to eq("Houston Dynamo")
   end
 
-  #league stats
+  it 'returns highest scoring visitor team' do
+    #Data has been mocked below since initial dummy data set did not have varying seasons.
+    fake_data = [
```

Comment:
awesome use of fake_data. probably can use a before block or let block in the beginning for smaller code